docs/user_guide/trace.md

```diff
@@ -317,3 +357,37 @@ The meaning of the trace timestamps is:
     scheduling.
 
   * BACKEND_OUTPUT: The tensor in the response of a backend.
+
+## Opentelemetry trace support
+
+Triton provides an option to generate and export traces 
+using [Opentelemetry APIs and SDKs](https://opentelemetry.io/). 
+
+To specify Opentelemetry mode for tracing, specify the `--trace-config` flag as follows:
+
+```
+$ tritonserver --trace-config=mode=opentelemetry --trace-config=opentelemetry,url=<endpoint> ...
+```
+
+### Opentelemetry trace APIs settings
```

Comment:
"Opentelemetry" -> "OpenTelemetry" here and everywhere.

Reply:
Updated